plugins/modules/nd_backup.py

```diff
@@ -28,38 +30,88 @@
   encryption_key:
     description:
     - The encryption_key for a backup file.
+    - A minimum of 8 alphanumeric characters is required.
```

Comment:
This is only true for state backup right?

Reply:
Yes